src/utils/static-log-level/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This crate is used to control static log level.
```

Comment:
To avoid forgetting to enable the static log level in release build, how about adding a compile-time check like this?

```rust
#[cfg(all(not(feature = "enabled"), not(debug_assertions)))]
compile_error!("must enable `static-log-level` in release build with `--features static-log-level`")
```

Reply:
I'd prefer moving static-log-level and static-link in a single crate. e.g. workspace-config with enable-static-link and enable-log-level feature.